    * nmudiff.1: close bold
    * README: s/bazaar/bzr, debpkg is below debi.
    * dscverify.pl: remove obsolete comment

diff -u -r devscripts-2.10.1/scripts/nmudiff.1 devscripts/scripts/nmudiff.1
--- devscripts-2.10.1/scripts/nmudiff.1	2007-02-26 00:16:03.000000000 +0100
+++ devscripts/scripts/nmudiff.1	2007-03-19 22:47:23.000000000 +0100
@@ -50,7 +50,7 @@
 devscripts configuration files; see below.
 .TP
 \fB\-\-from\fR \fIEMAIL\fR
-If using the sendmail (\fB\-\-no\-mutt) option, then the email to the
+If using the sendmail (\fB\-\-no\-mutt\fR) option, then the email to the
 BTS will be sent using the name and address in the environment
 variable DEBEMAIL.  If this is not set, then the variable EMAIL will
 be used instead.  These can be overridden using the \fB\-\-from\fR
diff -u -r devscripts-2.10.1/README devscripts/README
--- devscripts-2.10.1/README	2007-03-16 21:16:35.000000000 +0100
+++ devscripts/README	2007-03-19 22:47:57.000000000 +0100
@@ -59,7 +59,7 @@
   the .diffs and original files, though, so that the binaries and
   other files can be rebuilt if necessary.
 
-- debcommit [cvs | subversion | tla | bazaar | git]: Commits changes to 
+- debcommit [cvs | subversion | tla | bzr | git]: Commits changes to 
   cvs, svn, arch or git, using new entries in debian/changelog as the
   commit message. Also supports tagging Debian package releases.
 
@@ -70,7 +70,7 @@
   on the changes between source versions.
 
 - debi: Installs the current package by using the setuid root debpkg
-  script described above.  It assumes that the current package has
+  script described below.  It assumes that the current package has
   just been built (for example by debuild), and the .deb lives in the
   parent directory, and will effectively run dpkg -i on the .deb.  The
   ability to install the package with a very short command is very
diff -u -r devscripts-2.10.1/scripts/dscverify.pl devscripts/scripts/dscverify.pl
--- devscripts-2.10.1/scripts/dscverify.pl	2007-03-16 21:16:35.000000000 +0100
+++ devscripts/scripts/dscverify.pl	2007-03-19 23:06:16.000000000 +0100
@@ -85,10 +85,6 @@
     die xwarndie_mess @_;
 }
 
-# We retain the reference to Guy's directory on master even though
-# it won't be of use other than on master.  In this way, this program
-# may be installed on merkel without any changes being necessary,
-# and it is hardly likely to cause any ill effects elsewhere.
 sub get_rings {
     my @rings = @_;
     for (qw(/org/keyring.debian.org/keyrings/debian-keyring.gpg
